From 1249e3b862a119bdcba74f8f052222cc28de988d Mon Sep 17 00:00:00 2001
Message-Id: <1249e3b862a119bdcba74f8f052222cc28de988d.1604591893.git.crobinso@redhat.com>
From: Cole Robinson <crobinso@redhat.com>
Date: Thu, 5 Nov 2020 09:56:53 -0500
Subject: [PATCH] configure: Fix gio detection

Signed-off-by: Cole Robinson <crobinso@redhat.com>
---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index 2c3c69f118..bc06aa81e5 100755
--- a/configure
+++ b/configure
@@ -3499,7 +3499,7 @@ if $pkg_config --atleast-version=$glib_req_ver gio-2.0; then
     # with pkg-config --static --libs data for gio-2.0 that is missing
     # -lblkid and will give a link error.
     write_c_skeleton
-    if compile_prog "" "gio_libs" ; then
+    if compile_prog "" "$gio_libs" ; then
         gio=yes
     else
         gio=no
-- 
2.28.0

