diff -rup qemu-2.3.1/configure qemu-2.3.1.new/configure
--- qemu-2.3.1/configure	2015-08-11 20:19:07.000000000 +0100
+++ qemu-2.3.1.new/configure	2015-09-08 12:40:44.694091584 +0100
@@ -2323,7 +2323,7 @@ quorum_tls_libs=`$pkg_config --libs gnut
 if compile_prog "$quorum_tls_cflags" "$quorum_tls_libs" ; then
   qcow_tls=yes
   libs_softmmu="$quorum_tls_libs $libs_softmmu"
-  libs_tools="$quorum_tls_libs $libs_softmmu"
+  libs_tools="$quorum_tls_libs $libs_tools"
   QEMU_CFLAGS="$QEMU_CFLAGS $quorum_tls_cflags"
   quorum="yes"
 else
