Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
Index: qemu-2.9.0/monitor.c
===================================================================
--- qemu-2.9.0.orig/monitor.c
+++ qemu-2.9.0/monitor.c
@@ -983,10 +983,12 @@ static void qmp_unregister_commands_hack
 #ifndef TARGET_ARM
     qmp_unregister_command(&qmp_commands, "query-gic-capabilities");
 #endif
-#if !defined(TARGET_S390X) && !defined(TARGET_I386)
+#if 1 || !defined(TARGET_S390X) && !defined(TARGET_I386)
+    /*XXX emergency workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1467599 */
     qmp_unregister_command(&qmp_commands, "query-cpu-model-expansion");
 #endif
-#if !defined(TARGET_S390X)
+#if 1 || !defined(TARGET_S390X)
+    /*XXX emergency workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1467599 */
     qmp_unregister_command(&qmp_commands, "query-cpu-model-baseline");
     qmp_unregister_command(&qmp_commands, "query-cpu-model-comparison");
 #endif
